### DISCUSSION TITLE
Welcome to e-commerce Discussions!

### DISCUSSION THREAD

**Discussion by thuongtruong1009** [thuongtruong1009/e-shopee#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by suminoaapico** [comment]
Hi 
I need a USER Password
https://e-shopee.vercel.app/buyer/login
